src/polychord/clustering.f90

```diff
@@ -10,7 +10,7 @@ module KNN_clustering
     !! Points belong to the same cluster if they are in either of each others k
     !! nearest neighbor sets. 
     !!
-    !! The algorithm computes the k nearest neihbor sets from the similarity
```

Comment:
This is a hangover from #87, which I remember @williamjameshandley you asked me to include there